pkg/config/config.go

```diff
@@ -714,11 +714,11 @@ func (c *Config) CheckCgroupsAndAdjustConfig() {
 }
 
 func (c *Config) addCAPPrefix() {
-	for i, val := range c.Containers.DefaultCapabilities {
+	for i, val := range c.Containers.DefaultCapabilities.Get() {
 		if !strings.HasPrefix(strings.ToLower(val), "cap_") {
 			val = "CAP_" + strings.ToUpper(val)
 		}
-		c.Containers.DefaultCapabilities[i] = val
+		c.Containers.DefaultCapabilities.Values[i] = val
```

Comment:
IMO it would be better here to loop over Values here or create a new slice and then use Set() to replace the original.
Right now it kinda mixes both, as a reader I know must know that Get() returns Values otherwise this code is not safe.